project/Build.scala

```diff
@@ -1144,9 +1163,20 @@ object Build extends sbt.Build {
             val outFile = dir / "SourceMapTest.scala"
             IO.write(outFile, replaced.replace("0/*<testCount>*/", i.toString))
             Seq(outFile)
-          }
+          },
+
+        testFrameworks += new TestFramework("com.novocode.junit.JUnitFramework"),
```

Comment:
Yes, but only in the `testSuite` because there I can't use the `AutoPlugin`. Any other project that would use `enablePlugins(ScalaJSJUnitPlugin)` will not have to add the framework this way.